clone/cmd/sumdbclone/sumdbclone.go

```diff
@@ -33,13 +33,16 @@ import (
 )
 
 var (
-	height         = flag.Int("h", 8, "tile height")
-	vkey           = flag.String("k", "sum.golang.org+033de0ae+Ac4zctda0e5eza+HJyk9SxEdh+s3Ux18htTTAD8OuAn8", "key")
+	vkey           = flag.String("vkey", "sum.golang.org+033de0ae+Ac4zctda0e5eza+HJyk9SxEdh+s3Ux18htTTAD8OuAn8", "The verification key for the log checkpoints")
 	mysqlURI       = flag.String("mysql_uri", "", "URL of a MySQL database to clone the log into. The DB should contain only one log.")
 	writeBatchSize = flag.Uint("write_batch_size", 1024, "The number of leaves to write in each DB transaction.")
 	workers        = flag.Uint("workers", 4, "The number of worker threads to run in parallel to fetch entries.")
 )
 
+const (
+	height = 8
```

Comment:
nit: maybe `tileHeight` just to be explicit?

Reply:
Done.